### ISSUE TITLE
Pregunta 3

### ISSUE THREAD

**Issue by TarekHirmas** [IIC2413/Syllabus-2023-1#7]
Hola! En la pregunta 3, cuando consideran "a todas las personas que bucearon...", esto seria solamente buzos o buzos y guías?

Desde ya, muchas gracias

**Comment by TarekHirmas** [comment]
me di cuenta que en "Participó", por cada inmersión, deben por lo menos 2 columnas (la del buzo y la del guía), no? de ser así creo que la duda está resuelta

**Comment by RICARDOOVIEDO** [comment]
Efectivamente, cada buzo esta asociado a un guía. Por lo tanto, se consideran todas las personas.
